openthread_border_router/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.13.1
```

Comment:
I think it's worth bumping at least the minor version here given the character of the changes.